pkg/GAPJulia/JuliaInterface/src/convert.c

```diff
@@ -16,10 +18,16 @@ jl_value_t * julia_gap(Obj obj)
         return gap_box_gapffe(obj);
     }
     if (IS_JULIA_OBJ(obj)) {
-        return GET_JULIA_OBJ(obj);
+        BEGIN_GAP_SYNC();
+        result = GET_JULIA_OBJ(obj);
+        END_GAP_SYNC();
```

Comment:
Why do we need to wrap calls to `GET_JULIA_OBJ` into SYNC sections? We only read a single value from the JuliaObj GAP object here; and these objects are effectively read-only, i.e., they are never modified after being created.

Reply:
Ah, I hadn't realized that they were immutable, I think. This simplifies things.